cmd/api-resources.go

```diff
@@ -71,7 +71,7 @@ func getListObjectsV2Args(values url.Values) (prefix, token, startAfter, delimit
 
 	// The continuation-token cannot be empty.
 	if val, ok := values["continuation-token"]; ok {
-		if len(val[0]) == 0 {
+		if len(val) == 0 {
```

Comment:
This is not correct you need to make sure - the continuation token is not empty. 

`continuation-token` if it exists in `values` then it will always have one element i.e `len(val)` is never 0 when `continuation-token` is present in `values` map.

So the check you are changing is wrong. We are actually interested in `if len(val[0]) == 0` not `len(val) == 0` - we already know `len(val)` is >= 1 - so it is safe to use use `len(val[0])`